### ISSUE TITLE
Google Drive Folders don't appear

### ISSUE THREAD

**Issue by mjonstone** [sash04ek/afterlogic-google-drive-plugin#1]
Hello, 

Is Drive folders intended on being supported? As far as I can see from the code it doesn't have support for it. Can this be added?

**Comment by sash04ek** [comment]
Yes, Google Drive folders are fully supported. In fact, the code doesn't contain any special processing for folders specifically.
